src/AutoRest.CSharp/Common/Output/Expressions/Snippets.Bicep.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using AutoRest.CSharp.Common.Output.Expressions.KnownValueExpressions;
+using AutoRest.CSharp.Common.Output.Expressions.Statements;
+using AutoRest.CSharp.Common.Output.Expressions.ValueExpressions;
+using AutoRest.CSharp.Output.Models.Types;
+
+namespace AutoRest.CSharp.Common.Output.Models
+{
+    internal static partial class Snippets
+    {
+        public static class InvokeBicep
+        {
+            public static InvokeStaticMethodStatement AppendChildObject(
```

Comment:
Instead of an extra snippet here people can just call the instance method on the type provider directly